setup_requirements.py

```diff
@@ -46,6 +46,7 @@
     'plumpy==0.10.2',
     'circus==0.14.0',
     'tornado==4.5.3',  # As of 2018/03/06 Tornado released v5.0 which breaks circus 0.14.0
+    'bpython==0.17.1',
```

Comment:
Should this be a new standard requirement? And if so, remember to add it to rtd requirements

Reply:
Indeed, I wouldn't add bpython (we just reused the logic of Django, at least in some old version, that was allowing also for bpython if ipython was not there, but I'm not even sure of if bpython still exists...)

Reply:
It exists. I installed it to test this. I can remove the dependency but for me it doesn't make sense to try to load a shell in AiiDA (and also to explicitly mention it at the AiiDA cli arguments) when it is not part of the AiiDA dependencies.

Reply:
I would say it should be an extra requirement.